pkg/app/app_deploy.go

```diff
@@ -97,6 +97,9 @@ func (a *App) inspect() exec.CmdRunResult {
 
 	for _, dep := range a.app.Spec.Deploy {
 		switch {
+		case dep.Kapp != nil && dep.Kapp.Inspect == nil:
```

Comment:
lets turns this into a conditional under case dep.Kapp case (switch cases are only picking types of deployers eg kapp today)

Reply:
I thought of doing that but wanted to avoid the additional nesting, but it does makes sense to keep the behaviour consistent. Will do.

Reply:
Done.